server/player.lua

```diff
@@ -402,15 +465,15 @@ function QBCore.Player.CreatePlayer(PlayerData, Offline)
         self[fieldName] = data
     end
 
-    if self.Offline then
```

Comment:
Why was this changed?

Reply:
That was changed due the Offline logic changing so I could add it onto the database.
I've tested it all and works, not unrelated to pr, since it's needed to add the online field.

Reply:
For more context, the way it was synced and stored before didn't allow it to properly sync onto the database, and most of the times we want to know if player is online and not offline, so I figured since I was reworking the code I changed it to online, inside PlayerData, that way we can know specifically what character is online.
Old way would only let us know if the player was online but not the character itself.